src/Carousel.js

```diff
@@ -22,6 +22,7 @@ const propTypes = {
    * @default 'carousel'
    */
   bsPrefix: PropTypes.string,
+  as: PropTypes.element,
```

Comment:
`elementType` still, `element` is different. We can change in the whole project yeah if you are up for it